diff -ur c-ares-1.9.1.org/ares_private.h c-ares-1.9.1/ares_private.h
--- c-ares-1.9.1.org/ares_private.h	2011-12-19 23:58:22.000000000 +0200
+++ c-ares-1.9.1/ares_private.h	2013-01-26 01:12:06.268079847 +0200
@@ -74,11 +74,15 @@
 
 #else
 
-#define PATH_RESOLV_CONF        "/etc/resolv.conf"
+#ifndef SYSCONFDIR
+#define SYSCONFDIR "/etc"
+#endif
+
+#define PATH_RESOLV_CONF        SYSCONFDIR "/resolv.conf"
 #ifdef ETC_INET
-#define PATH_HOSTS              "/etc/inet/hosts"
+#define PATH_HOSTS              SYSCONFDIR "/inet/hosts"
 #else
-#define PATH_HOSTS              "/etc/hosts"
+#define PATH_HOSTS              SYSCONFDIR "/hosts"
 #endif
 
 #endif
